asus/zephyrus/ga402/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  services = {
+    asusd.enable = lib.mkDefault true;
```

Comment:
We still need to wait until the channel moved before merging this.

Reply:
All channels should have moved by now.